mesonbuild/modules/i18n.py

```diff
@@ -212,6 +212,7 @@ def merge_file(self, state: 'ModuleState', args: T.List['TYPE_var'], kwargs: 'Me
             install=kwargs['install'],
             install_dir=[kwargs['install_dir']] if kwargs['install_dir'] is not None else None,
             install_tag=install_tag,
+            description='Merging translations {}',
```

Comment:
```suggestion
            description='Merging translations for {}',
```

Reply:
"for"  because that file is effectively copied over with translations added in -- the action performed is adding the translations, not producing the desktop file or whatever.